library/src/main/java/com/pokegoapi/api/pokemon/Pokemon.java

```diff
@@ -170,9 +172,12 @@ public Result transferPokemon() throws LoginFailedException, RemoteServerExcepti
 	 * @return the boolean
 	 * @throws LoginFailedException  the login failed exception
 	 * @throws RemoteServerException the remote server exception
+	 * @throws InvalidCurrencyException the currency has exception
 	 */
-	public boolean canPowerUp() throws LoginFailedException, RemoteServerException {
-		return getCandy() >= getCandyCostsForPowerup();
+	public boolean canPowerUp() throws LoginFailedException, RemoteServerException, InvalidCurrencyException {
+		return getCandy() >= getCandyCostsForPowerup()
```

Comment:
@FabianTerhorst The exception was removed 2 minutes before your comment. Luckily, @mgorski-zlatan mentioned that a recent change has modified the PR without presenting an error. I am not sure why there had to be this elaborate claim a commit from a day ago did not account for a change 45 minutes ago without simply referencing the change.